l10n_ch_account_reconcile_esr/__manifest__.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# copyright 2019 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    'name': 'Switzerland - ESR account reconcile',
```

Comment:
Please use the english name "ISR".

We did it wrong in the past by using BVR everywhere. But the official term in english is ISR for In-payment Slip with Reference.